src/libstd/error.rs

```diff
@@ -89,7 +75,10 @@ pub trait Error: Debug + Display {
     /// }
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
-    fn description(&self) -> &str;
+    #[doc(hidden)]
```

Comment:
I don't think hiding this in the docs is a good idea. It's still a stable function so the docs need to include something for it. I suggest just changing the docs above instead.

Reply:
I assume you mean something like “if it exists it should be documented”. The point of this change is to try to make it as if the method did not exist anymore, except without breaking code that already implements or calls it.

Reply:
> “if it exists it should be documented”

Yeah, is that controversial?

> The point of this change is to try to make it as if the method did not exist anymore

You can't rewrite history. What do you expect people to do if they see this method used in code and want to find out what it does, or in fact why it's depreciated? I don't see the problem with just making it clear in the documentation that this method is deprecated and shouldn't be used any more.

Reply:
> is that controversial?

No, not that part.

> You can't rewrite history.

*Shrug* I don’t really care either way, this PR was only motivated by making sure that the outcome of last week’s libs meetings were not lost in a forgotten Dropbox Paper doc. At this point I’ll leave it to someone else in @rust-lang/libs to decide.